pkg/scheduler/plugins/predicates/predicates.go

```diff
@@ -328,7 +328,7 @@ func (pp *predicatesPlugin) OnSessionOpen(ssn *framework.Session) {
 	})
 
 	features := feature.Features{
-		EnableVolumeCapacityPriority:                 utilFeature.DefaultFeatureGate.Enabled(features.VolumeCapacityPriority),
+		//EnableVolumeCapacityPriority:                 utilFeature.DefaultFeatureGate.Enabled(features.VolumeCapacityPriority),
```

Comment:
In predicates, the `VolumeCapacityPriority` needs to be replaced by `StorageCapacityScoring` instead

Reply:
done.